ResortUI.java

```diff
@@ -67,12 +67,14 @@ private void listAllResort() {
 
     // provide the code here  
     private void listAllPasses() {
-
```

Comment:
Make sure this method is formatted correctly.

Reply:
okk